### ISSUE TITLE
Django support

### ISSUE THREAD

**Issue by jakob-o** [fellowshipofone/django-celery-transactions#13]
EDIT: By looking at the code it seems Django 1.6 to 1.8 is supported. If that's correct feel free to close the issue - unless you want to update the docs ;)

Hi,

it seems that this fork seems to be maintained in contrary to @bradleyayers original repository. I wondered which django versions are supported by this fork, since it is not clearly mentioned in the docs (started a discussion here https://github.com/bradleyayers/django-celery-transactions/issues/17).

Regards

Jakob

**Comment by jakob-o** [comment]
Followup question:
I am using celery 3.1.19, am I correct that the task decorator is using the configuration of the first occurrence of the celery app in contrast to the shared_task decorator as explained here https://groups.google.com/forum/#!topic/celery-users/XiSDiNjBR6k?

Thanks and Regards

Jakob

**Comment by fellowshipofone** [comment]
Hey, 

Sorry for the late reply, lost the notifications. Yes, Django 1.6 to 1.8 are supported. I'll started testing the 1.9 beta soon, but I know they are making changes to post-transactions hooks so it will most likely require more work.

I dropped support for 1.5 because Django dropped it too, and it was making the code too complex.

And yes, I took it over from the original branch which was not maintained, and took over the official PyPi repository.
